index.html

```diff
@@ -1,54 +1,77 @@
-      	<li><a title='About Us' href="about.html">About Us</a></li>
-        <li><a title='Join our mailing list' href="#">Mailing List</a>
-          <ul>
-            <li><a href="http://frodo.hserus.net/mailman/listinfo/ilugd" target="_blank" rel=external>Subscribe</a></li>
-            <li><a href="http://bit.ly/pydelhi-mailinglist-guidelines" target="_blank" rel=external >Guidelines</a></li>
-          </ul>
-        </li>
-        <li><a title='Other social links' href="#">Social</a>
-          <ul>
-              <li><a href="http://meetu.ps/c/33hSL/mrw5T/f" target="_blank" rel=external >MeetUp</a></li>
-            <li><a href="https://groups.google.com/forum/#!forum/ilugd/" target="_blank" rel=external >Google Group</a></li>
-            <li><a href="https://www.facebook.com/ILUGD/" target="_blank" rel=external >Facebook Page</a></li>
-            <li><a href="https://twitter.com/ilugdelhi" target="_blank" rel=external >Twitter</a></li>
-            <li><a href="https://t.me/joinchat/AAAAAEAc48wCHkUXViXBcg" target="_blank" rel=external >Telegram</a></li>
-            <li><a href="https://www.reddit.com/r/ILUGD/" target="_blank" rel=external >Reddit</a></li>
-            <li><a href="https://riot.im/app/#/room/#ilugd:matrix.org" target="_blank" rel=external >Matrix</a></li>
-          </ul>
-        </li>
-        <li><a href="http://github.com/ilugd" target="_blank" rel=external
-               title="Visit and collaborate with us on Github">Github</a>
-        </li>
-      </ul>
+  <div class="wrapper">
+
+    <section class="header">
+      <div class="content">
+        <div class="navbar">
+          <a class="link" href="#">About</a>
+          <div class="dropper">
+            <a class="link">Mailing list</a>
+            <div class="drop-wrap">
+              <div class="drop">
+                <a href="#">Subscribe</a>
+                <a href="#">Guidelines</a>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="branding">
+          <div class="logo"></div>
+          <div class="logotype">
+            !LUG-D
+          </div>
+        </div>
+        <p class="intro">
+          Welcome to the <br>India Linux Users Group Delhi.
+        </p>
+        <div class="social">
+          <a href="#"><i class="fa fa-meetup" aria-hidden="true"></i></a>
+          <a href="#"><i class="fa fa-google-plus" aria-hidden="true"></i></a>
+          <a href="#"><i class="fa fa-facebook-official" aria-hidden="true"></i></a>
+          <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+          <a href="#"><i class="fa fa-telegram" aria-hidden="true"></i></a>
+          <a href="#"><i class="fa fa-reddit" aria-hidden="true"></i></a>
+          <a href="#"><i class="fa fa-github" aria-hidden="true"></i></a>
+        </div>
+      </div>
+    </section>
+
+    <section class="meetup">
+      <div class="content">
+        <div class="section--next_event"></div>
```

Comment:
The meetup API is not working because the function is taking the value of the meetup name here. 
`<div class="section--next_event" data-meetup-group="ILUGDelhi"></div>`

Reply:
👍

Reply:
did you fix this @axios047 ? Let's get this merged to master ASAP. 

Sincere apologies for not contacting you sooner.